packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -271,10 +287,41 @@ const themedSharedStyles: ThemedStylesFn<ButtonThemeContract> = (theme) => ({
     },
     startIcon: {
         marginInlineEnd: theme.padding.small,
+        marginInlineStart: -theme.padding.small,
```

Comment:
**thought:** One thing that got me thinking is that `marginInlineStart` (and `end`) is not supported by Safari v14 (we currently have to support this version). I think it would be fine to keep it this way as we haven't gotten any reports about this, but it's worth keeping an eye on it.

Hopefully we'll update our support list to support Safari v15+ in the near future.